### DISCUSSION TITLE
Demo sites

### DISCUSSION THREAD

**Discussion by carolcoral** [liuzhihang/halo-theme-hao#575] [General]
如果你也在使用这个主题，可以把地址贴在这里。

@sunheyi6 https://shyblog.world/
@lqbby https://www.lqbby.com/
@carolcoral https://blog.cnkj.site/
@Akuns https://klog.club/
@Adyun https://blog.adyun.design/
@Xiaokai6880  https://ai.smallai.asia/
@xiaozhangStudent https://blog.xiaozhangstu.com/

**Comment by carolcoral** [comment]
@sunheyi6 https://shyblog.world/
@lqbby https://www.lqbby.com/
@carolcoral https://blog.cnkj.site/
@Akuns https://klog.club/
@Adyun https://blog.adyun.design/
@Xiaokai6880  https://ai.smallai.asia/

**Reply by ruibaby** [reply]
第一个不是我，是 @liuzhihang

**Reply by carolcoral** [reply]
已更新

**Comment by sun0225SUN** [comment]
delete me，我润回去 Hexo 了，哈哈

**Reply by carolcoral** [reply]
已处理

**Comment by SmallAi-API** [comment]
https://ai.smallai.asia

**Reply by carolcoral** [reply]
had updated

**Comment by xiaozhangStudent** [comment]
update 一下 https://blog.xiaozhangstu.com

**Reply by xiaozhangStudent** [reply]
@carolcoral

**Reply by carolcoral** [reply]
<img width="763" alt="image" src="https://github.com/chengzhongxue/halo-theme-hao/assets/8952687/d634d93f-017b-46ce-9ed1-e16267d2e90a">

**Reply by xiaozhangStudent** [reply]
已修复https://blog.xiaozhangstu.com/

**Comment by gfjz** [comment]


**Reply by gfjz** [reply]
高峰君主的网站

**Comment by wx-1124** [comment]
一心一意用hao主题，可以上个演示站点吗TAT
https://blog.nanshengwx.cn/

**Reply by carolcoral** [reply]
上

**Reply by carolcoral** [reply]
站点名字写上

**Reply by wx-1124** [reply]
楠笙

**Comment by atlavtis2010** [comment]
登逆 | https://denlee.cc
知识分享，记录成长

**Reply by carolcoral** [reply]
已加

**Comment by fyanjin** [comment]
申请添加
阿锦在线 https://www.ajinol.com
